src/archivist/fetcher/fullDomFetcher.js

```diff
@@ -86,7 +86,21 @@ export async function launchHeadlessBrowser() {
     return browser;
   }
 
-  browser = await puppeteer.launch({ headless: true });
+  const options = {
+    args: [],
+    headless: true,
+  };
+  if (process.env.http_proxy) {
+    options.args = [].concat(options.args, `--proxy-server=${process.env.http_proxy}`);
```

Comment:
Should this be uppercase? If not, could you explain why?
```suggestion
    options.args = [].concat(options.args, `--proxy-server=${process.env.HTTP_PROXY}`);
```